content/rs/references/cli-utilities/rladmin/node/enslave.md

```diff
@@ -0,0 +1,157 @@
+---
+Title: rladmin node enslave
+linkTitle: enslave
+description: Changes the resources of a node to replicas.
+weight: $weight
+alwaysopen: false
+toc: "true"
+headerRange: "[1-2]"
+categories: ["RS"]
+aliases:
+---
+
+`rladmin node enslave` changes the resources of a node to replicas.
+
+## `node enslave`
+
+Changes all of the node's endpoints and shards to replicas.
+
+``` sh
+$ rladmin node <id> enslave
+                [demote_node]
+                [retry_timeout_seconds]
```

Comment:
`retry_timeout_seconds` accepts an integer, right? If so, the other two commands need to be updated too.
```suggestion
                [retry_timeout_seconds] <seconds>
```